src/amcrest/event.py

```diff
@@ -193,21 +193,21 @@ def event_stream(self, eventcodes):
         except TypeError:
             timeout_cmd = (self._timeout_default, None)
         ret = self.command(
-            'eventManager.cgi?action=attach&codes=[{0}]'.format(eventcodes),
+            "eventManager.cgi?action=attach&codes=[{0}]".format(eventcodes),
```

Comment:
Just curious, any reason why replacing all ' with " ?

Reply:
black again...

Reply:
> black again...

I still don't get it :(

Reply:
black is a code formatter. It appears that hound-bot is now using it. Go back and look at the previous hound-bot reviews (click on "Show resolved") and you'll see what I mean.

Reply:
ah-ha! Thanks!